src/fastmcp/server/auth/oauth_proxy.py

```
@@ -2151,6 +2294,26 @@ async def _show_consent_page(
+        )  # type: ignore[union-attr]
+
+        # Extract domain from CIMD URL for display in consent screen
+        cimd_domain = None
+        if is_cimd_client:
+            try:
+                from urllib.parse import urlparse
+
+                parsed = urlparse(txn["client_id"])
+                cimd_domain = parsed.hostname
+            except Exception:
+                pass
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)